src/package.xml

```diff
@@ -9,4 +9,4 @@
 			<file path="Hotkey/widget/"/>
 		</files>
 	</clientModule>
-</package>
\ No newline at end of file
+</package>
```

Comment:
Version number should be updated to next major, Add note on which original version it was based